cmd/origin_token.go

```diff
@@ -129,6 +130,13 @@ func cliTokenCreate(cmd *cobra.Command, args []string) error {
 		args = append(args, audSlice...)
 	}
 
+	// If a custom issuer is provided, we need to add it to the claims. If there is none, we'll end up falling back to
+	// the issuer generated by config.go/GetServerIssuerURL()
```

Comment:
Where do we implement the falling back? I didn't find it in the `origin_token` file

Reply:
After some digging I think I figured out the logic, and it looks good to me

Reply:
For the sake of leaving thorough breadcrumbs in case anyone else winds up asking this (or things break in the future), the fallback happens in `token/token_create.go` on the call to `tokenConfig.CreateTokenWithKey()`. If the token config is instantiated without an issuer, the token defaults to `config.GetServerIssuerURL()`.